.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will build a package using Gradle and then publish it to GitHub Packages and the Maven Central Repository.
+# For more information see: https://github.com/actions/setup-java#publishing-using-gradle
+
+name: Publish package
+
+on:
+  release:
```

Comment:
Just for my understanding: this is only done when a new release is being made from GitHub, or also when a commit is tagged?

Reply:
Yes, type `created` will only be triggered when a non-draft release is created and published.